skills/verification-before-completion/SKILL.md

```
@@ -75,6 +75,16 @@ Skip any step = lying, not verifying
 
 ## Key Patterns
 
+**Configuration Changes:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)